docs/tutorial/part-one/index.md

```diff
@@ -348,41 +350,40 @@ npm install --global surge
 surge login
 ```
 
-Next, build your site by running the following command in the terminal at the root of your site (tip: make sure you're running this command at the root of your site, in this case in the hello-world folder, which you can do by opening a new tab in the same window you used to run `gatsby develop`):
+इसके बाद, अपनी साइट के रूट में टर्मिनल में निम्नलिखित कमांड चलाकर अपनी साइट का निर्माण करें (टिप: सुनिश्चित करें कि आप इस कमांड को अपनी साइट के रूट पर चला रहे हैं, इस मामले में हैलो-वर्ल्ड फ़ोल्डर में, जिसे आप उसी विंडो में एक नया टैब खोलकर  कर सकते हैं जिसमे आप `gatsby develop` चलाते थे:
 
 ```shell
 gatsby build
 ```
 
-The build should take 15-30 seconds. Once the build is finished, it's interesting to take a look at the files that the `gatsby build` command just prepared to deploy.
+बिल्ड को १५-३० सेकंड लेना चाहिए। एक बार बिल्ड समाप्त हो जाने के बाद, अब उन फाइलों पर एक नज़र रखना दिलचस्प है जो `gatsby build` कमांड केवल डिप्लाय करने के लिए तैयार करता हैं।
 
-Take a look at a list of the generated files by typing in the following terminal command into the root of your site, which will let you look at the `public` directory:
+अपनी साइट के रूट में टर्मिनल कमांड में निम्नलिखित टाइप करके उत्पन्न फ़ाइलों की सूची पर एक नज़र डालें, अब आपको `public` फोल्डर दिखेगा:
 
 ```shell
 ls public
 ```
 
-Then finally deploy your site by publishing the generated files to surge.sh.
+फिर आखिर में उत्पन्न फ़ाइलों को surge.sh पर प्रकाशित करके अपनी साइट को डिप्लाय करें।
 
 ```shell
 surge public/
 ```
 
-Once this finishes running, you should see in your terminal something like:
+एक बार जब यह समाप्त हो जाता है, तो आपको अपने टर्मिनल में कुछ इस तरह से दिखना चाहिए:
 
 ![Screenshot of publishing Gatsby site with Surge](surge-deployment.png)
 
-Open the web address listed on the bottom line (`lowly-pain.surge.sh` in this
-case) and you'll see your newly published site! Great work!
+नीचे पंक्ति मे सूची वेब पते को खोलें(`lowly-pain.surge.sh`) और आप अपनी नई प्रकाशित साइट देखेंगे! अच्छा कार्य!
 
-## ➡️ What’s Next?
+## ➡️ आगे क्या?
 
-In this section you:
+इस भाग में आपने:
 
-- Learned about Gatsby starters, and how to use them to create new projects
-- Learned about JSX syntax
-- Learned about components
-- Learned about Gatsby page components and sub-components
-- Learned about React “props” and reusing React components
+-  शुरुआत मे Gatsby स्टार्टर के बारे में सीखा, और नये प्रॉजेक्ट्स को बनाने के लिए उसका उपयोग कैसे करें
```

Comment:
` शुरुआत मे Gatsby स्टार्टर के बारे में सीखा, और नये प्रॉजेक्ट्स को बनाने के लिए उसका उपयोग कैसे करें` => `Gatsby स्टार्टर के बारे में और उसे नए प्रोजेक्ट बनाने में कैसे इस्तेमाल करें, सीखा`

how about this?

Reply:
how about ```और उसे नए प्रोजेक्ट बनाने में इस्तेमाल कैसे करें सीखा```